utils/abis/mint-abi.json

```diff
@@ -0,0 +1,13 @@
+[
```

Comment:
I didn't use it after all, but learned that this can be used instead of contract name when you call `hre.ethers.getContractAt(abiJsonObjectFromThisJson, address)` you can provide just a small part of the abi and call the function you need to call. Pretty useful for external contract communication. I decided to keep this abi, even though it's not used anywhere. WDYT?

Reply:
cool, it can be very useful - so great to know that 😎
I wonder whether a future version of us (or another developer) will understand why this file is here. I don't think we should keep it

Reply:
You're right, I got too excited about it ;)